cms/static/coffee/spec/views/course_info_spec.coffee

```diff
@@ -153,6 +153,19 @@ define ["js/views/course_info_handout", "js/views/course_info_update", "js/model
             it "does not remove existing course info on click outside modal", ->
                 @cancelExistingCourseInfo(false)
 
+            it "does not allow updates to be saved with an invalid date", ->
+                @courseInfoEdit.onNew(@event)
+                @courseInfoEdit.$el.find('input.date').val("Marchtober 40, 2048")
+                @courseInfoEdit.$el.find('input.date').change()
+                expect(@courseInfoEdit.$el.find('.save-button').hasClass("is-disabled")).toEqual(true)
```

Comment:
Extend the test in these ways:

1) Verify that Save button is enabled on initial load (assuming it default to current date).
2) After verifying that the button disables after entering an invalid date, change the date to something valid and make sure it enables the Save button.